src/main/java/org/aksw/simba/squirrel/frontier/impl/FrontierImpl.java

```diff
@@ -60,6 +52,27 @@ public UriQueue getQueue() {
      */
     protected GraphLogger graphLogger;
 
+
+    /**
+     * Indicates whether recrawling is active.
+     */
+    private boolean doesRecrawling;
+
+    /**
+     * The timer that schedules the recrawling.
+     */
+    private Timer timerRecrawling;
+
+    /**
+     * Time (in milliseconds) after which uris will be recrawled.
+     */
+    public static final long RECRAWL_TIME = 1000 * 60 * 60 * 24 * 7;
+
+    /**
+     * Time interval (in milliseconds) after which the check for outdated uris is performed.
+     */
+    private static final int TIMER_PERIOD = 1000 * 60 * 60;
```

Comment:
The same as above. One `static final` default value and a second value that can be configured.